Classes/common/CDTIncrementalStore.m

```diff
@@ -16,6 +16,14 @@
 #import "CDTFieldIndexer.h"
 #import "CDTISGraphviz.h"
 
+@interface CDTISBatchUpdateResult : NSPersistentStoreResult
+@property (nonatomic, strong) id result;
+@property (nonatomic, assign) NSBatchUpdateRequestResultType resultType;
```

Comment:
I suspected this and just confirmed that, at least, the compiler is happy with:
```objc
@interface NSBatchUpdateResult ()
@property (strong, readwrite) id result;
@property (readwrite) NSBatchUpdateRequestResultType resultType;
@end
```

I suppose you could also extend it with an initializer, but that could conflict in the future